### ISSUE TITLE
Test data leakage issue

### ISSUE THREAD

**Issue by george-kris** [zaixizhang/ANS-GT#5]
from your code, the train_dataset, valid_dataset, and test_dataset will random update in every update period T:
![image](https://github.com/zaixizhang/ANS-GT/assets/29527934/9cf86c8c-466e-45b3-a23f-c662b5e1e6a4)
How to ensure that without test data leakage issues? 
You customized a random partition protocol with 0.6/0.2/0.2 for train/valid/test, which is different from the standard protocol for Planetoid data (cora, citeseer, PubMed), and in your update period, you random split again, the test data may leakage to model training.

**Comment by zaixizhang** [comment]
Hi, thanks for your interest in our paper. In random_split, we used the fixed random seed so that the split is fixed in every update. Therefore, there will be no data leakage issues.
